tests/test_dem.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import os
+
+from umbral.dem import UmbralDEM
+from cryptography.exceptions import InvalidTag
+
+
+def test_encrypt_decrypt():
+    key = os.urandom(32)
+
+    dem = UmbralDEM(key)
+
+    plaintext = b'attack at dawn'
+
+    ciphertext0 = dem.encrypt(plaintext)
+    ciphertext1 = dem.encrypt(plaintext)
+
+    assert ciphertext0 != plaintext
+    assert ciphertext1 != plaintext
+
+    # Ciphertext should be different even with same plaintext.
+    assert ciphertext0 != ciphertext1
+
+    # Nonce should be different
+    assert ciphertext0[:12] != ciphertext1[:12]
```

Comment:
Perhaps for clarity you can use DEM_KEYSIZE and DEM_NONCE_SIZE to replace 32 and 12, respectively.

Reply:
And a comment explaining that the ciphertext format is `nonce + enc_data + tag`